operate/client/src/modules/utils/getClientConfig.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {z} from 'zod';
+
+const ClientConfigSchema = z.object({
```

Comment:
@juliasaienko Could you create a follow up PR to consume all client config flags from this new function?